lib/widgets/home_appbar_content.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+import '../constants/string_constants.dart';
+
+class HomeAppBarContent extends StatelessWidget {
+  final String srcImage;
+
+  HomeAppBarContent({
+    this.srcImage = StringConstants.defaultLogo,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+
+      ],
+    );
+  }
+}
```

Comment:
If you don't use this class, remove it please

Reply:
Done!